typings/parcel__core/index.d.ts

```diff
@@ -0,0 +1,6 @@
+type FixMeLater = any
+declare module "@parcel/core" {
+  export class Parcel {
+    constructor(options: FixMeLater)
+  }
+}
```

Comment:
RIP parcel types. They opted for Flow so we don't have types 😢

Reply:
I guess there's no way to transform Flow to TypeScript?

Reply:
I mean it's possible but not without some work unfortunately 😓 Not sure if you saw the [linked issue](https://github.com/parcel-bundler/parcel/issues/3912) in the description but it sounds like other people in the parcel community are asking for it so 🤞